### PR TITLE
Fix query string parsing

### LINKED ISSUE #31: Process stuck in Firefox
When doing `ztoken` and firefox set as default browser the process gets stuck and the HTML only shows **Redirecting**.

By the looks of it the include JavaScript does not extract the Token from the URL properly.

```url
http://localhost:8081/#access_token=....
```

in chrome its been casted to

```url 
http://localhost:8081/?access_token=
```

### PR DESCRIPTION
**Pull request by prayerslayer**
Work around [bug in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1101653) by rewriting logic to ES5.

Fixes #31

### CODE REVIEW

File: zign/api.py [hunk 1]
```diff
@@ -73,9 +81,7 @@
         <p>Your browser does not support Javascript! Please enable it or switch to a Javascript enabled browser.</p>
     </noscript>
     <p>Redirecting...</p>
-    <p id="error">
-        Error: no access token.
-    </p>
+    <p id="error"></p>
```
Comment (gargravarr) [inline zign/api.py:84]: We can remove this too, right?

Comment (prayerslayer) [reply zign/api.py:84]: Nope, needed to show error message :) Will be filled by `displayError`.

**Comment by gargravarr** [comment]
Tested with Firefox, still not redirecting.

**Comment by prayerslayer** [comment]
👍

**Pull request merged**